hal/src/tron/hal_platform_config.h

```diff
@@ -4,10 +4,10 @@
 #include "platforms.h"
 
 #define HAL_PLATFORM_NCP (1)
-#define HAL_PLATFORM_USART2 (1)
 #define HAL_PLATFORM_NCP_AT (0)
 #define HAL_PLATFORM_CELLULAR (0)
 // #define HAL_PLATFORM_CELLULAR_SERIAL (HAL_USART_SERIAL2)
+#define HAL_PLATFORM_USART3 (1)
```

Comment:
@scott-brust For P2 both USART2 and USART3 should be available, right?

Reply:
@avtolstoy Yes, Serial2 mostly conflicts with SPI1 and Serial3 mostly conflicts with SPI but both are available. 
On MSoM, USART3/Serial3 is dedicated to the cell modem.

Reply:
Yup, and this is a P2/Photon2-specific configuration file, so let's enable both.

Reply:
Ah I see, we should have 
```
#define HAL_PLATFORM_USART2 (1)
#define HAL_PLATFORM_USART3 (1)
```
ie enable all 3 usarts on P2/Photon2

Reply:
I'd say yeah. @technobly @XuGuohui ?

Reply:
I see this is already defined in `hal_platform_rtl8721x_config.h`:
```
#define HAL_PLATFORM_USART2 (1)
```
Ill just remove the commented out line about cellular serial since its not relevant for P2/Photon2